webpack-aot.config.js

```diff
@@ -8,14 +8,9 @@ webpackConfig.module.rules = [
   loaders.tslint,
   loaders.ts,
   loaders.html,
-  { test: /\.css$/, use: 'raw-loader', include: /node_modules/ },
```

Comment:
Just saw this. I think this was needed for an AoT fix with component libraries. @winkerVSbecks??